web/client/src/library/pages/docs/Docs.tsx

```diff
@@ -1,79 +1,106 @@
-      {models.size === 0 ? (
-        <div className="p-4 flex flex-col w-full h-full overflow-hidden">
-          <div className="flex justify-center items-center w-full h-full">
-            <div className="center">
-              <h3 className="text-lg mb-4">
-                Still Waiting For Models To Load...
-              </h3>
-              <p>It should take few more seconds.</p>
-              <p>
-                If it takes too long, probably, there is a problem with API
-                response.
-              </p>
-              <p>Check if the server is running and refresh.</p>
-            </div>
-          </div>
-        </div>
-      ) : (
-        <div className="p-4 flex flex-col w-full h-full overflow-hidden">
-          {isArrayNotEmpty(filtered) && (
-            <SearchList<ModelSQLMeshModel>
-              list={filtered}
-              size={EnumSize.lg}
-              searchBy="index"
-              displayBy="displayName"
-              to={model => `${EnumRoutes.IdeDocsModels}/${model.name}`}
-              isFullWidth={true}
+    <Page
+      sidebar={
+        <SourceList
+          key={location.pathname}
+          by="displayName"
+          byName="displayName"
+          to={EnumRoutes.IdeDocsModels}
+          items={list}
+          types={list.reduce(
+            (acc: Record<string, string>, it) =>
+              Object.assign(acc, {
+                [it.name]: getModelNodeTypeTitle(
+                  it.type as LineageNodeModelType,
+                ),
+              }),
+            {},
+          )}
+          listItem={({ to, name, description, text, disabled = false }) => (
+            <SourceListItem
+              to={to}
+              name={name}
+              text={text}
+              description={description}
+              disabled={disabled}
             />
           )}
-          <SplitPane
-            className="flex w-full h-full overflow-hidden mt-8"
-            sizes={[25, 75]}
-            minSize={0}
-            snapOffset={0}
-          >
-            <div className="py-4 w-full">
-              {models.size > 0 && (
-                <SourceList
-                  models={models}
-                  filter={filter}
-                  setFilter={setFilter}
+        />
+      }
+      content={
+        <Container.Page>
+          {models.size === 0 ? (
+            <div className="p-4 flex flex-col w-full h-full overflow-hidden">
+              <div className="flex justify-center items-center w-full h-full">
+                <div className="center">
+                  <h3 className="text-lg mb-4">
+                    Still Waiting For Models To Load...
+                  </h3>
+                  <p>It should take few more seconds.</p>
+                  <p>
+                    If it takes too long, probably, there is a problem with API
+                    response.
+                  </p>
+                  <p>Check if the server is running and refresh.</p>
```

Comment:
Should we say this stuff at all?
```suggestion
                  <p>It should take a few more seconds.</p>
                  <p>
                    If it takes too long, there is probably a problem with the API
                    server.
                  </p>
                  <p>Check if the server is running and refresh.</p>
```